### PR TITLE
Fixes #3710

### LINKED ISSUE #3710: Post.english.php - Error in the text
Last pull request #3701 

Post.english.php line 131 and 132 is 2x locked. Is that right? Or it should not be unlocked in line 132
line 131 ...topic was locked, line 132 ...topic was **?un?**locked
```
$txt['error_topic_locked'] = 'Warnung - while you were typing the topic was locked. Please check the "Lock this topic" box under "Attachments and other options" below if you do not wish to undo this action.';
$txt['error_topic_unlocked'] = 'Warnung - while you were typing the topic was locked. Please uncheck the "Lock this topic" box under "Attachments and other options" below if you do not wish to undo this action.';
```

### PR DESCRIPTION
**Pull request by live627**
Signed-off-by: John Rayes <live627@gmail.com>

### CODE REVIEW

**Review by colinschoen — APPROVED** [review]
LGTM

**Comment by live627** [comment]
Quick question:  How did you get that phrase to trigger the approvals status?

**Comment by colinschoen** [comment]
It isn't automatic. You can approve changes when reviewing the file diffs. 

screen shot 2016-12-09 at 17 46 49

**Pull request merged**
